app/models/book.rb

```diff
@@ -1,2 +1,12 @@
 class Book < ApplicationRecord
+    has_many :reservations
```

Comment:
in the same time a book cannot have many reservations. please take a look at this again!

Reply:
A book can have many reservations, for example in the next month, the book can be rented out to multiple people. This validation you are talking about, that a book can't be rented out multiple times at the same time, we'll have to validate later.